### PR TITLE
Add user experience section

### PR DESCRIPTION
**Pull request by ann0see**
**Short description of changes**

Adds a small section to the CONTRIBUTING.md file which links to the style and tone guide

**Context: Fixes an issue?**

Although this was mainly about documentation, having an unified style (and raising awareness on it) would benefit the UX of Jamulus.

**Does this change need documentation? What needs to be documented and how?**

Yes. The same has to be applied on the documentation repo. It's even more important there.

**Status of this Pull Request**

Working implementation; final wording checks.

**What is missing until this pull request can be merged?**
Two reviews.

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [ ] I tested my code and it does what I want
- [x] My code follows the [style guide](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#source-code-consistency) 
- [ ] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I've filled all the content above

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -45,7 +45,10 @@ Please try to avoid breaking any of them by introducing platform-specific code.
 Keep all code compatible to C++11.
 ### Dependencies
 If your code requires new dependencies, be sure that those are available on all supported platforms and that the introduction of those dependencies has been discussed and accepted.
-
+### User experience
+Jamulus is used by people all over the world with different backgrounds and knowledge. Features and language should be user focused.
+Language wise, the Jamulus project has a [style and tone guide](https://jamulus.io/contribute/Style-and-Tone) which you should follow to remain consistent.
+Feature wise, think what your least tech-savvy friend would think about your change: Would it result in confusion or ambiguity?
```
Comment (gilgongo) [inline CONTRIBUTING.md:51]: The appeal to those who know less than yourself is a bit uncomfortable beause it implies superiority over them. I would neutralise it to:

```suggestion
Features should be usable in the sense that they act as expected to somebody who does not have a technical background. 
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -45,7 +45,10 @@ Please try to avoid breaking any of them by introducing platform-specific code.
 Keep all code compatible to C++11.
 ### Dependencies
 If your code requires new dependencies, be sure that those are available on all supported platforms and that the introduction of those dependencies has been discussed and accepted.
-
+### User experience
+Jamulus is used by people all over the world with different backgrounds and knowledge. Features and language should be user focused.
+Language wise, the Jamulus project has a [style and tone guide](https://jamulus.io/contribute/Style-and-Tone) which you should follow to remain consistent.
```
Comment (gilgongo) [inline CONTRIBUTING.md:50]: Can we shorten this to:

```suggestion
The Jamulus project has a [style and tone guide](https://jamulus.io/contribute/Style-and-Tone) which you should follow to remain consistent.
```

**Comment by ann0see** [comment]
Android is failing…

**Review by gilgongo — APPROVED** [review]
Looks good to me.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -45,7 +45,10 @@ Please try to avoid breaking any of them by introducing platform-specific code.
 Keep all code compatible to C++11.
 ### Dependencies
 If your code requires new dependencies, be sure that those are available on all supported platforms and that the introduction of those dependencies has been discussed and accepted.
-
+### User experience
+Jamulus is used by people all over the world with different backgrounds and knowledge. Features and language should be user focused.
+The Jamulus project has a [style and tone guide](https://jamulus.io/contribute/Style-and-Tone) which you should follow to remain consistent.
+Features should be usable in the sense that they act as expected to somebody who does not have a technical background.
```
Comment (pljones) [inline CONTRIBUTING.md:51]: Blank lines before and after section?

Comment (ann0see) [reply CONTRIBUTING.md:51]: Depends on what we prefer.

Comment (ann0see) [reply CONTRIBUTING.md]: Ok. Done

**Review by softins — APPROVED** [review]
Looks fine to me.
